### PR TITLE
Refactor/77/update 1 12 2

### PR DESCRIPTION
**Pull request by Syndlig**
also converted a few block recipes over to the new forge recipe format. haven't tested them - wanted to get this pushed before I do any more work - but they _should_ work.

### CODE REVIEW

File: src/main/java/com/gb/cornucopia/cookery/cuttingboard/BlockCuttingBoard.java [hunk 1]
```diff
@@ -39,57 +42,44 @@ public BlockCuttingBoard() {
...
-				? BOARD_AABB[0]
-				: BOARD_AABB[1];
+		return (state.getValue(FACING).getAxis() == EnumFacing.Axis.X) ? BOARD_AABB[0] : BOARD_AABB[1];
 	}
 
 	@Override
-	public IBlockState getStateFromMeta(final int meta) {
+	public IBlockState getStateFromMeta(int meta) {
 		// must be horizontal
 		final EnumFacing facing = EnumFacing.getFront(meta).getAxis() == EnumFacing.Axis.Y
```
Comment (Liruilos) [inline src/main/java/com/gb/cornucopia/cookery/cuttingboard/BlockCuttingBoard.java]: EnumFacing has a boolean method isHorizonal(), would that help here?

File: src/main/java/com/gb/cornucopia/cookery/mill/TileEntityMill.java [hunk 2]
```diff
@@ -45,6 +45,15 @@ public int getSizeInventory() {
 		return 9;
 	}
 
+	public boolean isEmpty() {
```
Comment (Liruilos) [inline src/main/java/com/gb/cornucopia/cookery/mill/TileEntityMill.java]: isn't this a bit confusing also being called isEmpty? I am not even sure what it is used for yet though 😆

File: src/main/java/com/gb/cornucopia/cookery/stove/ContainerStove.java [hunk 3]
```diff
@@ -15,19 +15,19 @@
 	public final TileEntityStove stoveInventory;
 	private int cook_time;
 	private int burn_time;
-	private int butn_time_max;
```
Comment (Liruilos) [inline src/main/java/com/gb/cornucopia/cookery/stove/ContainerStove.java]: lol, nice catch. also i read this as butt time

File: src/main/java/com/gb/cornucopia/proxy/ClientProxy.java [hunk 4]
```diff
@@ -17,7 +17,6 @@
 	@Override
 	public void preInit(final FMLPreInitializationEvent e) {
 		super.preInit(e);
-		InvModel.register();
```
Comment (Liruilos) [inline src/main/java/com/gb/cornucopia/proxy/ClientProxy.java]: are you doing registry differently now?

Comment (Syndlig) [reply src/main/java/com/gb/cornucopia/proxy/ClientProxy.java]: Yep! The way it's handled in newer versions of Forge is you use `@EventBusSubscriber` to register the `RegistrationHandler` class, [which you can see here](https://github.com/team-soas/cornucopia/pull/80/files#diff-788dd1c7503c783b95b1d13041489f91R58).

**Pull request merged**